cms/static/js/certificates/views/certificate_editor.js

```diff
@@ -140,45 +138,12 @@ function($, _, Backbone, gettext,
             );
         },
 
-        setOrgLogoPath: function(event) {
-            // Updates the indicated model field (still requires persistence on server)
-            if (event && event.preventDefault) { event.preventDefault(); }
-            var org_logo_path = this.$('.org-logo-input').val();
-            this.model.set(
-                'org_logo_path', org_logo_path,
-                { silent: true }
-            );
-            this.$('.current-org-logo img.org-logo').attr('src', org_logo_path);
-        },
-
         setValues: function() {
             // Update the specified values in the local model instance
             this.setName();
             this.setDescription();
             this.setCourseTitle();
-            this.setOrgLogoPath();
             return this;
-        },
-
-        uploadLogoImage: function(event) {
-            event.preventDefault();
-            var upload = new FileUploadModel({
-                title: gettext("Upload organization logo."),
-                message: gettext("Maximum logo height should be 125px."),
-                mimeTypes: ['image/png', 'image/jpeg']
-            });
-            var self = this;
-            var modal = new FileUploadDialog({
-                model: upload,
-                onSuccess: function(response) {
-                    var org_logo_path = response.asset.url;
-                    self.model.set('org_logo_path', org_logo_path);
-                    self.$('.current-org-logo img.org-logo').attr('src', org_logo_path);
-                    self.$('.current-org-logo').show();
-                    self.$('input.org-logo-input').attr('value', org_logo_path);
-                }
-            });
-            modal.show();
         }
     });
     return CertificateEditorView;
```

Comment:
@asadiqbal08 you should remove loading of uploads model and view at the top of this.
`js/models/uploads`,    `js/views/uploads`